sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/implementation/HmacAuthenticationPolicy.java

```diff
@@ -127,13 +129,14 @@ public HmacAuthenticationPolicy(AzureKeyCredential clientCredential) {
         headers.put(CONTENT_HASH_HEADER, contentHash);
         String utcNow = OffsetDateTime.now(ZoneOffset.UTC)
             .format(HMAC_DATETIMEFORMATTER_PATTERN);
+        headers.put(X_MS_DATE_HEADER, utcNow);
         headers.put(DATE_HEADER, utcNow);
```

Comment:
Is this still needed?

Reply:
Yes. We want to have log entries on service side to see if LinkedIn proxies actually modified Date header.